code/game/machinery/doors/poddoor.dm

```diff
@@ -47,8 +63,8 @@ var/list/poddoors = list()
 	if ((src.density && (stat & NOPOWER) && !( src.operating )))
```

Comment:
I know this isn't technically your code but could you remove all the "src."s here, they're unnecessary.